pkg/apis/operator/v1alpha1/common.go

```diff
@@ -95,6 +97,11 @@ type KComponentStatus interface {
 	// SetVersion sets the currently installed version of the component.
 	SetVersion(version string)
 
+	// GetManifests gets the url links of the manifests
+	GetManifests() string
+	// SetManifests sets the url links of the manifests
```

Comment:
What's the format?

Reply:
Good question:
single url: just the full url link or url link with ${version}
multiple urls: concatenate them together with comma as delimiter.

Reply:
The manifests in status is set automatically by the operator. Users do not care it.

Reply:
Interesting. So getManifests is a way for the Operator to expose what versions it 'knows about'?

Reply:
I think I'd feel more comfortable about this interface as `[]string` rather than "`string` with internal comma separation". It seems easier for computers to parse, too.

Reply:
This one under KComponentStatus just indicate what the actual paths are for manifests.
If the manifests consist of multiple links, which can be local paths, can be remote paths, etc, they will be concatenated together, with the delimiter `,`.

Reply:
Ah, ok. So in the status, the Version shown will always be the selected one.

Do we still need a GetVersion in the spec, though?